docs/plugins/spectate.rst

```diff
@@ -2,16 +2,47 @@ spectate
 ========
 
 .. dfhack-tool::
-    :summary: Automatically follow exciting dwarves.
+    :summary: Automatically follow productive dwarves.
     :tags: fort interface
-    :no-command:
 
 Usage
 -----
 
 ::
 
     enable spectate
+    spectate set <setting> <value>
+    spectate enable|disable <feature>
 
-The plugin will automatically switch which dwarf is being followed periodically,
-preferring dwarves on z-levels with the highest job activity.
+
+When enabled, the plugin will automatically switch which dwarf is being
+followed periodically, preferring dwarves on z-levels with the highest
+job activity.
+
+Changes to plugin settings will be saved per world. Whether the plugin itself
+is enabled or not is not saved.
+
+Examples
+--------
+
+``spectate``
+    The plugin reports its configured status.
+
+``spectate enable auto-unpause``
+    Enable the spectate plugin to automatically dismiss pause events caused
+    by the game. Siege events are one example of such a game event.
+
+``spectate set tick-interval 50``
+    Set the tick interval the followed dwarf can be changed at back to its
+    default value.
+
+Features
+--------
+:focus-jobs:     Toggle whether the plugin should always be following a job. (default: disabled)
+:auto-unpause:   Toggle auto-dismissal of game pause events. (default: disabled)
+:auto-disengage: Toggle auto-disengagement of plugin through player intervention while unpaused. (default: disabled)
+
+Settings
+--------
+:tick-threshold: Set the plugin's tick interval for changing the followed dwarf.
+                 Acts as a maximum follow time when used with focus-jobs enabled. (default: 50)
```

Comment:
From some testing, a changing dwarves every 50 ticks seems very jarring. I found somewhere between 1000 and 10,000 to be more comfortable (at 100 FPS).

Of course, this is affected greatly by FPS -- maybe it would be better to measure seconds instead of ticks?

Reply:
and how many seconds was that? I think we should at least stick with the ticks for now, and add an issue to switch to seconds in the future if the discussion bears it out. What do you think?

Reply:
sounds good